Sources/ProjectDescription/TestAction.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+public struct TestAction: Equatable, Codable {
+    public let targets: [TestableTarget]
+    public let arguments: Arguments?
+    public let configurationName: String
+    public let coverage: Bool
+    public let codeCoverageTargets: [TargetReference]
+    public let preActions: [ExecutionAction]
+    public let postActions: [ExecutionAction]
+
+    public init(targets: [TestableTarget] = [],
```

Comment:
only difference between these two initialisers are one takes `configurationName` and other `config`, we can wrap these parameters under an `enum`.

Reply:
That's a good suggestion, although I wouldn't do it as part of this PR.